.github/workflows/ci.yaml

```
@@ -0,0 +1,48 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
```

Use GH macro (I didn't know it existed when we first started moving the CI)
```suggestion
      - $default-branch
```